tests/integrations/test-integration.php

```diff
@@ -147,19 +141,435 @@ public function test__is_active_returns_false_when_integration_is_not_active():
 		$this->assertFalse( $integration->is_active() );
 	}
 
-	public function test__set_vip_config_function_throws_error_if_integration_is_not_active(): void {
+	public function test__is_active_via_vip_returns_false_if_empty_config_is_provided(): void {
```

Comment:
- Moved methods from `test-integration-config.php` class to here.
- Updated methods to allow multiple configs.